### ISSUE TITLE
Improve the accessibility of our site

### ISSUE THREAD

**Issue by AndrewVSutherland** [roed314/seminars#362]
We should make whatever changes we can to make our site more accessible, e.g. to facilitate screen readers, deal with any contrast or font-sizing issues, etc...  It would be great if people could comment on specific changes they think should be made, as this issue is very vague as it stands.

It probably makes sense to address issue #25 first, since some of the changes needed to make the site more mobile friendly may also help with accessibility.
